bundles/binding/org.openhab.binding.fritzbox/README.md

```diff
@@ -1,5 +1,11 @@
 # Fritz!Box Binding
 
+## Prerequisites
+
+Note that this binding requires you to enable telnet on your FRITZ!Box, for which, due to security reasons, support has been dropped in firmware version 06.25 and later.
+
+Instead, you may use the [Fritzbox binding (using TR06 protocol)](./fritzboxtr0641).
```

Comment:
This link doesn't work.

Reply:
The only kind of link that works is an absolute link. Not sure what you mean by "with the versioning".
I'd argue for linking to the docs page at https://www.openhab.org/addons/bindings/fritzboxtr0641/